README.md

```diff
@@ -169,13 +169,15 @@ Table of contents:
 #### Server-Side Rendering
 * [Angular 2 Universal Repository (GitHub)](https://github.com/angular/universal)
 
-#### Material Design
+#### 
```

Comment:
Yes, the headline read `#### Material Design`, now it's just `Design`. It should be as it was before it was changed, as in `#### Material Design`. Thanks.